src/applications/vaos/appointment-list/components/cards/express-care/ExpressCareCard.jsx

```diff
@@ -43,25 +43,19 @@ export default function ExpressCareCard({
       <ExpressCareStatus appointment={appointment} />
       <div className="vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
         <div className="vads-u-flex--1 vads-u-margin-right--1 vaos-u-word-break--break-word">
-          <dl className="vads-u-margin--0">
-            <dt className="vads-u-font-weight--bold vads-u-display--block">
-              Your contact details
-            </dt>
-            <dd>
-              <Telephone contact={getPatientTelecom(appointment, 'phone')} />
-              <br />
-              {getPatientTelecom(appointment, 'email')}
-            </dd>
-          </dl>
+          <h4 className="vads-u-font-size--h4">Your contact details</h4>
```

Comment:
I think this one needs to take into account the `headingLevel` prop. For some uses of this component, we pass `2` as the heading level, which means this header would need to be an `h3` (though the class name would stay the same).

Reply:
Done